src/ResourceManagement/OperationalInsights/Microsoft.Azure.Management.OperationalInsights/Generated/Models/SavedSearch.cs

```diff
@@ -0,0 +1,132 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.OperationalInsights.Models
+{
+    using System.Linq;
+
+    /// <summary>
+    /// Value object for saved search results.
+    /// </summary>
+    [Microsoft.Rest.Serialization.JsonTransformation]
+    public partial class SavedSearch
+    {
+        /// <summary>
+        /// Initializes a new instance of the SavedSearch class.
+        /// </summary>
+        public SavedSearch() { }
+
+        /// <summary>
+        /// Initializes a new instance of the SavedSearch class.
+        /// </summary>
+        /// <param name="category">The category of the saved search. This helps
+        /// the user to find a saved search faster. </param>
+        /// <param name="displayName">Saved search display name.</param>
+        /// <param name="query">The query expression for the saved search.
+        /// Please see
+        /// https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-search-reference
+        /// for reference.</param>
+        /// <param name="version">The version number of the query lanuage. Only
+        /// verion 1 is allowed here.</param>
+        /// <param name="id">The id of the saved search.</param>
+        /// <param name="etag">The etag of the saved search.</param>
+        /// <param name="tags">The tags attached to the saved search.</param>
+        public SavedSearch(string category, string displayName, string query, long version, string id = default(string), string etag = default(string), System.Collections.Generic.IList<Tag> tags = default(System.Collections.Generic.IList<Tag>))
+        {
+            Id = id;
+            Etag = etag;
+            Category = category;
+            DisplayName = displayName;
+            Query = query;
+            Version = version;
+            Tags = tags;
+        }
+
+        /// <summary>
+        /// Gets the id of the saved search.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "id")]
+        public string Id { get; private set; }
```

Comment:
yes. it is a proxy resource living under workspace (a tracked resource). I can change the swagger to make this a proxy resource too.